.github/workflows/playwright-integration-tests-mysql.yml

```diff
@@ -14,17 +14,18 @@
 
 name: MySQL Playwright Integration Tests
 on:
+  pull_request:
```

Comment:
Hey @TeddyCr  - Do we need any constraints here on which file changes to run this CI or which paths to ignore ?

Reply:
yep, I just set it like that to test it on this PR 😊